main.go.tmpl

```diff
@@ -128,12 +128,15 @@ components:
       {{- end -}}
     {{- end}}
 
+{{- $operations := dict }}
 paths:
 {{- range $_, $service := .Services -}}
   {{- range $_, $method := .Methods}}
   {{- $deprecated := index $method.Annotations "deprecated" }}
   /rpc/{{$service.Name}}/{{$method.Name}}:
-    post:
+    post: {{- if exists $operations $method.Name }}
+      operationId: {{$service.Name}}{{$method.Name}} {{ else }}
+      operationId: {{$method.Name}}{{ set $operations $method.Name 1 }} {{- end }}
```

Comment:
> The id MUST be unique among all operations described in the API. The operationId value is case-sensitive. 

Why not use service+method name right away, e.g. `{{$service.Name}}-{{$method.Name}}`?

Reply:
I guess you're right. I thought it would be more readable this way, but on second thought, making the naming consistent is better.

Reply:
Ok done :)